lib/screens/home/home_screen.dart

```diff
@@ -85,6 +85,13 @@ class _HomeScreenState extends ConsumerState<HomeScreen> {
                     HomePagesWidget(
                       surveys: surveys,
                       currentPage: _currentPage,
+                      onNextButtonPressed: () {
```

Comment:
@Thieurom Next time, please move all the action functions outside the build functions. It will reduce the LOC in each function and make it more readability 🙏